interview-questions.md

```diff
@@ -4,55 +4,64 @@ Answer the following questions. First, without external resources. Challenge you
 
 
 
 4. What is the difference between console.log() and return?
 
-  Your answer:
+  Your answer: Console.log() just displays to the screen what you tell it to. Return is used at the end of function to define what the output or result of the function is.
 
-  Researched answer:
+  Researched answer: console. log() - logs out information in console. return just return from the function without processing further code or you can say, A function immediately stops at the point where return is called. (https://stackoverflow.com/questions/21020608/difference-between-console-log-and-return-in-javascript)
 
 
 
 5. In regards to functions, what is an argument?
 
-  Your answer:
+  Your answer: An argument, also known as a parameter is an input into a function. It is a value passed into the function that will be used, evaluated, or manipulated inside the function.
 
-  Researched answer:
+  Researched answer: 
+  Function parameters are the names listed in the function definition. Function arguments are the real values passed to (and received by) the function. (https://www.w3schools.com/js/js_function_parameters.asp)
 
 
 
-6. Give a brief description of proper pair programming techniques. What are the roles of each person?
 
-  Your answer:
 
-  Researched answer:
+6. Give a brief description of proper pair programming techniques. What are the roles of each person? 
+
+  Your answer: Proper pair programming techniques include communication between a pair of programmers to complete a portion of code. The pair includes a driver and a navigator. The driver does the actual coding and the navigator helps with the logic and reviewing the code for typos or errors.
+
+  Researched answer: With pair programming, two software developers work on one computer, collaborating on the same design, algorithm, code, or test. The programmer at the keyboard is usually called the “driver”, the other, also actively involved in the programming task but focusing more on overall direction is the “navigator”; it is expected that the programmers swap roles every few minutes or so. Pair programming is an important technique for developing higher quality code, faster while also reducing risk and spreading knowledge in an organization. (https://www.agilealliance.org/glossary/pairing/)
 
 
 
 7. What is something we did in class this week you found helpful?  
 
-  Your answer:
+  Your answer: I enjoyed the ice breaker / team building activities to help to get to know other cohort mates. I also found the stretch activities the most helpful to challenging my knowledge. 
```

Comment:
Awe yeah! Glad you enjoyed the team building activities and super stoked you like the stretch activities! 🤩